src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKeyCreationParameters.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    ///     Settings to be applied to a CNG key before it is finalized.
```

Comment:
On MSDN we have this as "Contains advanced properties for key creation.", which doesn't introduce confusion around the word "finalized".